src/test/rustdoc/intra-doc/extern-lang-item-impl.rs

```diff
@@ -0,0 +1,9 @@
+// Reexport of a structure that derefs to a type with lang item impls having doc links in their
+// comments. The doc link points to an associated item, so we check that traits in scope for that
```

Comment:
I think you forgot to add doc links?

Reply:
No, the doc links are in the standard library on the `impl f64 { ... }`.

Reply:
Hmm, can you please make this test stand-alone? We don't want to make this dependent on the content of the stdlib docs.

Reply:
Ok.